gen/calc.py

```diff
@@ -410,8 +418,8 @@ def validate_os_type(os_type):
             'aws': {
                 'must': {'rexray_config_contents': yaml.dump(AWS_REXRAY_CONFIG)},
             },
-            'empty': {
-                'must': {'rexray_config_contents': yaml.dump('')},
+            'disabled': {
```

Comment:
The name of this `rexray_config_method` value isn't mentioned in [Marathon docs](https://mesosphere.github.io/marathon/docs/external-volumes.html), so this change shouldn't be disruptive.